cmd/history.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+
+	"github.com/Benchkram/errz"
+	"github.com/puppetlabs/wash/api/client"
+	cmdutil "github.com/puppetlabs/wash/cmd/util"
+	"github.com/puppetlabs/wash/config"
+	"github.com/spf13/cobra"
+)
+
+func historyCommand() *cobra.Command {
+	historyCmd := &cobra.Command{
+		Use:   "history [<id>]",
+		Short: "Prints the wash command history, or journal of a particular item",
+		Args:  cobra.MaximumNArgs(1),
+	}
+
+	historyCmd.RunE = toRunE(historyMain)
+
+	return historyCmd
+}
+
+func printJournalEntry(index string) error {
```

Comment:
Yeah, I think a separate `historyEntry` type in the `activity` package would be cleaner.